pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -469,6 +469,17 @@ def test_to_datetime_tz(self, cache):
         with pytest.raises(ValueError, match=msg):
             pd.to_datetime(arr, cache=cache)
 
+    @pytest.mark.parametrize('cache', [True, False])
```

Comment:
Thanks for the explanation. Could you also parameterize over the `error` keyword since that was relevant, also including a value that would hit the error handling (i.e. `arr = [ts_string_1] * 5 + [ts_string_2] * 5 + ['foo']`)

Reply:
I have added a separate test using `hypothesis` to ensure that `pd.to_datetime()` produces equivalent results regardless of whether `cache` is `True` or `False`.